Nomination pool members cannot participate in OpenGov (#4924)
w3f/polkadot-wiki: docs/learn/learn-opengov.md

You can easily delegate your votes to somebody else using the
[Polkadot Delegation Dashboard](https://delegation.polkadot.network/). See
[this video tutorial](https://www.youtube.com/watch?v=RapBYZc5ZPo) to learn how to use the

> dashboard. **If you are a nomination pool member, you cannot participate in Governance with bonded
> tokens, nor does the pool admin as funds are held in a

Reviewer: ```suggestion
dashboard. **If you become a nomination pool member or a pool admin, you cannot participate in Governance with the bonded tokens in the pool, as they are held in a
```
